### ISSUE TITLE
Option to show full text on cards

### ISSUE THREAD

**Issue by arnaudspotfire** [hski-github/spotfire-kanban#4]
Would be nice to be able to switch between a compact 1 line layout (as it is today) to an expanded layout showing full text.

**Comment by hski-github** [comment]
The full text is in the tooltip. Is that okay? Or do you mean ability to expand for all cards? Then cards would have different heights, which would give long text more visual focus. But maybe a config about the general height of the card?

**Comment by arnaudspotfire** [comment]
Yes showing the full text in the tooltip definitely helps and I believe the mod is releasable as it is. I just thought it would be nice to have an expanded mode where all the cards would expand to show full text. This would mean that the heights of the cards would depends on the texts lengths.
It could be accessible from single button somewhere in the viz area (top right for instance). This button would be shown when the user is in editing mode and disappear when in viewing mode.
